filebeat/tests/system/test_prospector.py

```diff
@@ -164,9 +164,6 @@ def test_exclude_files(self):
             lambda: self.output_has(lines=1),
             max_timeout=15)
 
-        # TODO: Find better solution when filebeat did crawl the file
-        # Idea: Special flag to filebeat so that filebeat is only doing and
-        # crawl and then finishes
         filebeat.check_kill_and_wait()
```

Comment:
@urso not yet, but we had the comment in at least 10 times. I still left it in once so we remember.